sdks/python/apache_beam/ml/inference/sklearn_loader.py

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import abc
+import enum
+import pickle
+import sys
+from dataclasses import dataclass
+from typing import Any
+from typing import Iterable
+from typing import List
+
+import joblib
+import numpy
+
+import apache_beam.ml.inference.api as api
+import apache_beam.ml.inference.base as base
+import sklearn_loader
+from apache_beam.io.filesystems import FileSystems
+
+
+class SerializationType(enum.Enum):
+  PICKLE = 1
+  JOBLIB = 2
+
+
+class SKLearnInferenceRunner(base.InferenceRunner):
```

Comment:
Silly question, but for the sake of consistency and ease of use, what should the naming convention be for the different frameworks? For example, technically `PyTorch` has capital P and T, but in my implementation, I use `Pytorch` for simplicity. (I can change it though)

And for `Scikit-learn`, it's often abbreviated as `sklearn` or `Sklearn`, without the capital K or L. Should we change it to `SklearnInferenceRunner`?

Reply:
Looking at sklearn in particular it is always lower case.

Elsewhere in Beam we do have consecutive capitals like:
PValue
ReadFromTFRecord

I think for PyTorch you should just follow what you think users would expect (WRT naming).